Serializer/Normalizer/AbstractExceptionNormalizer.php

```diff
@@ -44,17 +44,17 @@ public function __construct(ExceptionValueMap $messagesMap, $debug)
     /**
      * Extracts the exception message.
      *
-     * @param \Exception $exception
+     * @param \Throwable $throwable
      * @param int|null   $statusCode
      *
      * @return string
      */
-    protected function getExceptionMessage(\Exception $exception, $statusCode = null)
+    protected function getMessageFromThrowable(\Throwable $throwable, $statusCode = null)
     {
-        $showMessage = $this->messagesMap->resolveException($exception);
+        $showMessage = $this->messagesMap->resolveThrowable($throwable);
```

Comment:
I can't find the `resolveThrowable()` method here:
https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Util/ExceptionValueMap.php

How it should work?

Reply:
It's only present in 2.x: https://github.com/FriendsOfSymfony/FOSRestBundle/blob/2.7/Util/ExceptionValueMap.php

The code for 3.0 was fixed in #2112.